app/stories/shared.stories.jsx

```diff
@@ -615,49 +614,6 @@ story('Tooltip')
         </Tooltip>
     ))
 
-const toolbarActions = {
```

Comment:
All stories are now in `$shared/components/Toolbar/Toolbar.stories.jsx`. Moved and improved. 💪